.github/workflows/e2e.yaml

```diff
@@ -18,7 +18,9 @@ name: E2E
 
 on:
   pull_request:
+    # Do not run this workflow when and only when `*.md` files are modified
     paths:
+      - '**'
```

Comment:
without `**`, GitHub Actions won't run these workflows kinda of misleading

Reply:
Got it.